docs/templates/how-to-template.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Start with a verb ending in -ing
+content_type: how-to
+
+# Optional values. Uncomment any that apply to your document.
+
+# no_version: true # Disables the version selector dropdown. Set this on pages that belong to doc sets without versions like /konnect/.
+# alpha: true # Labels the page as alpha; adds a banner to the top of the page.
+# beta: true # Labels the page as beta; adds a banner to the top of the page.
+---
+
+Add an introduction paragraph here. Good introductions explain who this how-to guide is for and what this guide will help the user accomplish.
+
+Keep the conceptual information to a minimum; we can assume the user already has some knowledge of the concepts. You can also add links to explanation articles if needed.
+
+<!-- See https://documentation.divio.com/how-to-guides/ for more info about how to write a how-to -->
+<!-- See the following examples of how-to documentation:
+* https://docs.konghq.com/gateway/latest/kong-enterprise/analytics/reports/
+* https://docs.konghq.com/gateway/latest/kong-manager/auth/ldap/service-directory-mapping/
+* https://docs.konghq.com/gateway/latest/plugin-development/custom-entities/
+-->
+<!-- Remove these comments once you are done writing -->
+
+## Prerequisites <!-- Optional -->
+
+<!-- Write prerequisites as a bulleted list. Only list prerequisites if they are application level prerequisites. -->
+<!-- If it isn't a "getting started" topic, we can assume our products are already installed. -->
+<!-- Don't prescribe Konnect or Gateway role permissions. -->
+
+* First prerequisite
+* Second prerequisite
+
+## Task section <!-- Header optional if there's only one task section in the article -->
```

Comment:
```suggestion
## Task section <!-- Header optional if there's only one task section in the article -->
```
Is that common? Should we encourage people to have more than one section?

Reply:
I think the case could be made that if you need multiple sections, you probably need to split it into multiple docs. 

I have done multiple sections in task-based topics before, like when you're doing an end-to-end workflow where you install and then configure the thing. So I guess it's used sometimes? But I don't know if we want to encourage it or not.